src/mutations/item.ts

```diff
@@ -115,6 +115,13 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
     mutationFn: (item) => Api.editItem(item.id, item, queryConfig),
     // newItem contains only changed values
     onMutate: async (newItem: Partial<Item>) => {
+      newItem = {
+        ...newItem,
+        name: newItem.name?.trim()
+      };
+
+      console.log(newItem);
```

Comment:
Remove this console. Maybe let's not mutate the argument directly, create a new variable.